### PR TITLE
feat: Add Confluent Cloud Schema Registry to ASSESS

### PR DESCRIPTION
**Pull request by OUeland**

### CODE REVIEW

**Review by sasjo — APPROVED** [review]

File: radar/data_management/confluent_schema_registry.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  [Confluent Cloud Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/)
+  is a central repository with a RESTful interface for developers to define standard schemas and register
+  applications to enable compatibility. Schema Registry is available as a
+  software component of Confluent Platform or as a managed component of
+  Confluent Cloud.
+rationale: |
+  Develop using standard schemas
+  Store and share a versioned history of all standard schemas, and validate
+  data compatibility at the client level.Schema Registry supports Avro,
+  JSON and Protobuf serialization formats.
```
Comment (Tasseri) [inline radar/data_management/confluent_schema_registry.yaml:17]: This is more of a description.

File: radar/data_management/confluent_schema_registry.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  Reduce operational complexity
+  Schema Registry reduces operational complexity in the application
+  development cycle, because it eliminates the need for complex coordination
+  among developers. Need to add a new column to a downstream database?
+  You don’t need an involved change process and at least 5 meetings to
+  coordinate 20 teams.
```
Comment (Tasseri) [inline radar/data_management/confluent_schema_registry.yaml:24]: I understand what you are trying to point out here, could you rephrase it in a less provocative way?

**Review by Tasseri — COMMENTED** [review]
Good add with a lot of good metadata. Rationale could be a bit sharper and concrete see my comments.
We usually recommend that you add a list of things to assess also. For reference you can look at the Looker assess blip:
https://tech-radar.extendaretail.com/entries/looker.html

I add these as comments as I don't see it necessary for Approval but it would raise the blip from good to great.

File: radar/data_management/confluent_schema_registry.yaml (hunk 2 above)
Comment (OUeland) [reply radar/data_management/confluent_schema_registry.yaml:24]: Certainly. The text is from the web-pages of Schema Registry. I didn't author any texts myself, but I certainly can.
